model/DataBaseAccess.php

```diff
@@ -177,7 +178,28 @@ public function addMultiplePermissions(array $permissionData)
                 }
             }
         }
-        $this->getPersistence()->insertMultiple(self::TABLE_PRIVILEGES_NAME, $insert);
+
+        $this->getLogger()->debug(
+            "Processing {count} permission inserts in {chunks} chunks",
+            [
+                'count' => count($insert),
+                'chunks' => ceil(count($insert)/self::INSERT_CHUNK_SIZE)
+            ]
+        );
+
+        foreach(array_chunk($insert, self::INSERT_CHUNK_SIZE) as $index => $batch) {
+            $this->getLogger()->debug(
+                "Processing chunk {index}/{total} with {items} ACL entries",
+                [
+                    'index' => $index + 1,
+                    'total' => ceil(count($insert)/self::INSERT_CHUNK_SIZE),
```

Comment:
Use separate variable for `count($insert)` instead of counting on every iteration.

Reply:
Missed that, changed in 0c6e230eaa4b7ffc1a0225ae00c07c1dd8942614 Thanks!